### ISSUE TITLE
Add support for array of arrays?

### ISSUE THREAD

**Issue by maharjun** [SmokinCaterpillar/pypet#40]
The array of array data format is a very commonly used format especially in the context of spiking neural networks (for example while storing spike data). HDF5 (or at-least h5py) seems to have a procedure for storing data of this type. See [here](http://docs.h5py.org/en/latest/special.html#arbitrary-vlen-data).

Note that I'm specifically referring to arrays of arrays containing a single data type. (That single data type may itself be an array thereby allowing recursive array-of-array data).

However, the last I checked, there was no support in Pypet to store data of this format, which is rather unfortunate as now the only option is to pickle the object thereby making it impossible to read in a HDF5 viewer.

In short It would be simply awesome if this data-type was supported.

**Comment by SmokinCaterpillar** [comment]
Hey maharjun, thanks for your remark. Do you have any idea if the arrays of arrays is supported by PyTables?

**Comment by maharjun** [comment]
The last I checked there was support for it in the form of `VLArrays`. See below for the link to the documentation.

[http://www.pytables.org/usersguide/libref/homogenous_storage.html#the-vlarray-class]()

**Comment by SmokinCaterpillar** [comment]
Thanks, this could work. Unfortunately, I am quite busy until mid of May, but then I'll take a look at it :-)

**Comment by SmokinCaterpillar** [comment]
no support for vlarray yet
